parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.kt

```diff
@@ -24,76 +25,98 @@ import android.content.Context
  * Accessed as a singleton. Maintains a queue of pageview events in memory and periodically
  * flushes the queue to the Parse.ly pixel proxy server.
  */
-public interface ParselyTracker {
+public object ParselyTracker {
+
+    private const val DEFAULT_FLUSH_INTERVAL_SECS = 60
+    private var instance: ParselyTrackerInternal? = null
+
+    private fun ensureInitialized(): ParselyTrackerInternal {
+        return instance ?: run {
+            throw ParselyNotInitializedException();
+        }
+    }
+
     public val engagementInterval: Double?
+        @JvmStatic
+        get() = ensureInitialized().engagementInterval
+
     public val videoEngagementInterval: Double?
+        @JvmStatic
+        get() = ensureInitialized().videoEngagementInterval
+
     public val flushInterval: Long
+        @JvmStatic
+        get() = ensureInitialized().flushInterval
 
-    public fun engagementIsActive(): Boolean
-    public fun videoIsActive(): Boolean
+    @JvmStatic
+    public fun engagementIsActive(): Boolean = ensureInitialized().engagementIsActive()
 
+    @JvmStatic
+    public fun videoIsActive(): Boolean = ensureInitialized().videoIsActive()
+
+    @JvmStatic
+    @JvmOverloads
```

Comment:
Sorry for one more change in this PR @oguzkocer . I've added `@JvmOverloads` annotations so the API will be nicer to work for Java consumers
![image](https://github.com/Parsely/parsely-android/assets/5845095/ae3f9c33-392a-43ff-a328-b063d10e0c12)

I thought I added it already before, but can't find a place. It'll be easier to track such changes after #101 is merged, which I plan to make ready for review after this PR.